dockerfiles/init/manifests/codenvy.env

```diff
@@ -106,6 +127,21 @@
 #     configure this as we manage it with the CLI.
 #CODENVY_SWARM_NODES=172.17.0.1:2375
 
+# Docker Host
+#     How Codenvy will connect to the Docker host if CODENVY_DOCKER_DAEMON__URL is not 
+#     set. This value can be provided on the command line as well. If this value is
+#     not set, then Codenvy will use the volume mount of unix:///var/run/docker.sock.
+#DOCKER_HOST=tcp://localhost:2375
+
+# External Docker IP Address
+#     The external IP address used by browsers to connect to workspace containers that
+#     are running on a Docker network. After Codenvy has created a workspace runtime,
+#     those Docker containers must be reachable by remote browsers. B 
+#     This will generally work if the browser and the workspace are on the same network.
+#     However, if the browser and the workspaces are on different networks, then you
+#     may need to set this value to the external IP address of your host or Docker to
+#     make workspaces reachable.
+#CHE_DOCKER_IP_EXTERNAL=NULL
```

Comment:
Maybe it is needed to mention that it works with single node only? Also As far as I can see it only works if codenvy is running on the same host as WSs